api/tokens_test.go

```diff
@@ -363,3 +364,199 @@ func TestAPI_RefreshHandler(t *testing.T) {
+			adminUserGlobalSignOutFunction func(signOutInput *cognitoidentityprovider.AdminUserGlobalSignOutInput) (*cognitoidentityprovider.AdminUserGlobalSignOutOutput, error)
+			httpResponse                   int
+		}{
+			{
+				// 200 response from Cognito - 202 from identity api
+				func(userInput *cognitoidentityprovider.ListUsersInput) (*cognitoidentityprovider.ListUsersOutput, error) {
+					users := mock.BulkGenerateUsers(3, nil)
+					return users, nil
+				},
+				func(signOutInput *cognitoidentityprovider.AdminUserGlobalSignOutInput) (*cognitoidentityprovider.AdminUserGlobalSignOutOutput, error) {
+					return &cognitoidentityprovider.AdminUserGlobalSignOutOutput{}, nil
+				},
+				http.StatusAccepted,
+			},
+		}
+		for _, tt := range signOutAllUsersTests {
+			m.ListUsersFunc = tt.listUsersFunction
+			m.AdminUserGlobalSignOutFunc = tt.adminUserGlobalSignOutFunction
+			r := httptest.NewRequest(http.MethodGet, usersEndPoint, nil)
+
+			successResponse, errorResponse := api.SignOutAllUsersHandler(ctx, w, r)
+			So(successResponse.Status, ShouldEqual, tt.httpResponse)
+			So(errorResponse, ShouldBeNil)
+		}
+	})
+}
+
+func TestSignOutAllUsersHandlerInternalServerError(t *testing.T) {
+	var ctx = context.Background()
+
+	api, w, m := apiSetup()
+
+	Convey("Testing users global signout - handler", t, func() {
+		signOutAllUsersTests := []struct {
+			listUsersFunction              func(userInput *cognitoidentityprovider.ListUsersInput) (*cognitoidentityprovider.ListUsersOutput, error)
+			adminUserGlobalSignOutFunction func(signOutInput *cognitoidentityprovider.AdminUserGlobalSignOutInput) (*cognitoidentityprovider.AdminUserGlobalSignOutOutput, error)
+			httpResponse                   int
+		}{
+			{
+				// 500 response from Cognito's ListUsers API endpoint
+				func(userInput *cognitoidentityprovider.ListUsersInput) (*cognitoidentityprovider.ListUsersOutput, error) {
+					awsErrCode := "InternalErrorException"
+					awsErrMessage := "Something strange happened"
+					awsOrigErr := errors.New(awsErrCode)
+					awsErr := awserr.New(awsErrCode, awsErrMessage, awsOrigErr)
+					return nil, awsErr
+				},
+				func(signOutInput *cognitoidentityprovider.AdminUserGlobalSignOutInput) (*cognitoidentityprovider.AdminUserGlobalSignOutOutput, error) {
+					return &cognitoidentityprovider.AdminUserGlobalSignOutOutput{}, nil
+				},
+				http.StatusInternalServerError,
+			},
+		}
+		for _, tt := range signOutAllUsersTests {
+			m.ListUsersFunc = tt.listUsersFunction
+			m.AdminUserGlobalSignOutFunc = tt.adminUserGlobalSignOutFunction
+			r := httptest.NewRequest(http.MethodGet, usersEndPoint, nil)
+
+			successResponse, errorResponse := api.SignOutAllUsersHandler(ctx, w, r)
+			So(successResponse, ShouldBeNil)
+			So(errorResponse.Status, ShouldEqual, tt.httpResponse)
+		}
+	})
+}
+
+func TestSignOutAllUsersGoRoutine(t *testing.T) {
+
```

Comment:
Seems to be an inconsistent blank line here

Reply:
👍